Lib/email/feedparser.py

```diff
@@ -424,12 +702,11 @@ def boundarymatch(line):
                 defect = errors.StartBoundaryNotFoundDefect()
                 self.policy.handle_defect(self._cur, defect)
                 self._cur.set_payload(EMPTYSTRING.join(preamble))
-                epilogue = []
                 for line in self._input:
                     if line is NeedMoreData:
                         yield NeedMoreData
                         continue
-                self._cur.epilogue = EMPTYSTRING.join(epilogue)
+                self._cur.epilogue = ''
```

Comment:
Note: I kept the behavior here the same, but I'm not actually certain whether it's correct. The previous code appeared as though it intended to assign the remainder of the message to the epilogue, but then did not.

So, this data is discarded. This is the only place where we discard the rest of the message like this. It's out of the scope of this PR, and it's very narrow in scope, but it's interesting.

Reply:
I think it's just to make sure that the attribute is initialized.